src/gtest/test_checkblock.cpp

```diff
@@ -446,6 +446,33 @@ TEST(ContextualCheckBlock, CoinbaseCommunityReward) {
     block.vtx[0] = CTransaction(mtx);;
     EXPECT_TRUE(ContextualCheckBlock(block, state, &indexPrev));
 
+    //Exceed the LastCommunityRewardBlockHeight
+    CChainParams params = Params();
+    int exceedHeight=params.GetConsensus()._deprecatedGetLastCommunityRewardBlockHeight()+1;
```

Comment:
You can avoid defining params variable and call Params().GetConsensus()... in line 451